pages/events.html

```diff
@@ -135,6 +140,30 @@ <h2 id="2023">2023</h2>
 		</div>
 
 		<div class="events-fulllist">
+			<a id="gdcmeetup-2023"></a>
+			<div class="card base-padding">
+				<h3>Godot GDC Meetup @ GitHub HQ</h3>
+
+				<img src="/storage/app/media/events/gdc-2023.jpg" alt="Godot GDC Meetup banner">
+				
+				<p>Feels good to be back! To continue with our tradition, thanks to our friends over at GitHub we are hosting the third annual Godot GDC Meetup.</p>
```

Comment:
```suggestion
				<p>Feels good to be back! To continue with our tradition, thanks to our friends over at GitHub we are holding the third annual Godot GDC Meetup.</p>
```

So it doesn't conflict with the next line where GitHub is hosting.